src/hooks/itemTag.ts

```diff
@@ -41,16 +43,18 @@ export default (queryConfig: QueryClientConfig, queryClient: QueryClient) => {
         if (!ids) {
           throw new UndefinedArgument();
         }
-        return Api.getItemsTags(ids, queryConfig).then((data) => List(data));
+        return Api.getItemsTags(ids, queryConfig).then((data) =>
+          convertJs(data),
+        );
       },
-      onSuccess: async (tags) => {
+      onSuccess: async (tags: List<RecordOf<ItemTag>[]>) => {
```

Comment:
Maybe this should be a list of list? I think it'll become confusing if we have a normal array in the middle.